### PR TITLE
[all hosts] Rework hub page to be more scenario/intent focused 

### PR DESCRIPTION
**Pull request by AlexJerabek**
Our hub page has higher-than-expected bounce rates and exit rates. This PR removes some of the links to pages with lower clicks. and promotes some scenarios, with the intention to substitute newer scenarios as they come online.

### CODE REVIEW

File: docs/index.yml [hunk 1]
```diff
@@ -23,34 +23,49 @@ highlightedContent:
...
-      title: Test and experiment with APIs using Script Lab
-      url: overview/explore-with-script-lab.md
-    # Card 8
-    - itemType: reference
-      title: Office JavaScript API reference documentation
-      url: /javascript/api/overview
+      title: Samples for Office Add-ins
+      url: overview/office-add-in-code-samples.md
 
+productDirectory:
```
Comment (samantharamon) [inline docs/index.yml:34]: I understand that the `title` attribute is [optional](https://review.learn.microsoft.com/help/platform/navigation-how-create-hub-page?branch=main#product-directory-section-multi-link), but should we consider adding one to the `productDirectory` section to match the `conceptualContent` section? Apart from the addition of the icon to the `productDirectory` cards, these sections share the same format (versus that of the top-level section). Not a deal-breaker, but thought it was worth asking.

**Review by samantharamon — APPROVED** [review]
Changes look good! Added a minor question on formatting.

File: docs/index.yml (hunk 1 above)
Comment (AlexJerabek) [reply docs/index.yml:34]: Good point. I agree that a title there would help draw attention. I'll try to think of something fitting, but I'm open to suggestions.

**Review by davidchesnut — APPROVED** [review]
This looks much better focused!

**Pull request merged**